CHANGELOG.md

```diff
@@ -4,6 +4,87 @@ This is the changelog file of the sw360 project. It starts with the first releas
 
 https://github.com/sw360/sw360portal/releases
 
+## sw360-14.0.0-M1
+
+This tag covers many corrections/bug fixes after the 13.3 release.
+
+This release provides features, muliple bug fixes for release 13.3, for example, new REST endpoints, new functions in the UI and changelog enable/disable from sw360.properties.
+
+### Migrations
+
+For this version, no database migration is necessary.
```

Comment:
There is a migration script for 
-  `e3d8122a` feat(ProjectUI): Add new values to Obligation status